src/components/core/base/VisualizationLoadingHOC.tsx

```diff
@@ -354,12 +354,32 @@ export function visualizationLoadingHOC<
                     return Promise.reject(new Error("Unknown projectId"));
                 }
 
-                const title = exportTitle ? escapeFileName(exportTitle) : "Untitled";
+                const {
+                    format,
+                    includeFilterContext,
+                    mergeHeaders,
+                    showFilters,
+                    title: customTitle,
+                } = exportConfig;
+
+                const title: string = escapeFileName(customTitle || exportTitle) || "Untitled";
+
+                const exportConfigRequest: IExportConfigRequest = {
+                    format,
+                    mergeHeaders,
+                    title,
+                };
+                if (includeFilterContext) {
+                    exportConfigRequest.showFilters = dataSource.getAfm().filters;
+                } else if (showFilters) {
```

Comment:
do we plan to deprecate this direct filters definition for UI SDK components? If yes, there should be some warning/info

Reply:
removed the backward compatibility's code